src/features/dashboard/components/Dialogs/DeleteAppDialog/index.tsx

```diff
@@ -20,42 +22,42 @@ const DeleteAppDialog = ({ appId, onClose }: TDeleteAppDialogProps) => {
     [onClose],
   );
 
-  const actionButtons: TModalActionButton[] = useMemo(
-    () => [
-      {
-        id: 0,
-        text: 'Yes, delete',
-        color: 'primary',
-        onClick: () => {
-          deleteApp(appId);
-          onClose();
-        },
-      },
-      {
-        id: 1,
-        text: 'Cancel',
-        color: 'secondary',
-        onClick: () => {
-          onClose();
-        },
-      },
-    ],
-    [appId, deleteApp, onClose],
-  );
+  const handleResize = () => {
+    setIsMobile(window.innerWidth < 720);
+  };
+
+  useEffect(() => {
+    window.addEventListener('resize', handleResize);
+    return () => {
+      window.removeEventListener('resize', handleResize);
+    };
+  }, []);
```

Comment:
@agrim-deriv look for this hook `useDeviceType` instead.

Reply:
It provides mobile, tablet and desktop so just use it accordingly.

Reply:
Hey @shafin-deriv , understood. Made the change. 
Thank you.